bundles/org.openhab.binding.lcn/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -277,6 +277,13 @@
 		<item-type>Contact</item-type>
 		<label>Binary Sensor</label>
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
+		<config-description>
+			<parameter name="invertOpenClosed" type="boolean">
+				<label>Invert Open/Closed</label>
+				<description>Per default lcn binary sensor state 0 corresponds to "closed" while 1 corresponds to "open". Use this parameter to invert that logic.</description>
```

Comment:
```suggestion
				<description>Per default a binary sensor state 0 corresponds to "closed" while 1 corresponds to "open". Use this parameter to invert that logic.</description>
```

Reply:
I mentioned "**lcn** binary sensor state" explicitly because "0" and "1" refer to their use in the lcn/pck's binary sensor state status message `Bx<w>` with w=0.255. Nevertheless íf you see a problem with this wording I can change it.

Reply:
Since this is the LCN binding, it don't need to be repeated.